features/importing/work_import.feature

```diff
@@ -64,6 +64,26 @@ Feature: Import Works
     Then I should see "Preview"
       And I should see "2010-01-11"
 
+  Scenario: Importing a new multichapter work with backdating
+    Given basic tags
+    And the following activated user exists
+      | login          | password    |
+      | cosomeone      | something   |
+    And I am logged in as "cosomeone" with password "something"
+    And I set my time zone to "UTC"
+    When I go to the import page
+    And I fill in "urls" with
+         """
+         http://rebecca2525.dreamwidth.org/3506.html
```

Comment:
Given that this feels like it is fixing a BOT right now, so not to worry immediately. But shouldn't we use the shiny new adt test account ?